shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/HikariConnectionPool.java

```diff
@@ -41,7 +43,8 @@ static HikariConfig constructHikariConfig() {
         String serverName = ApplicationContext.getProperty("DB_URL", "");
         String dbName = ApplicationContext.getProperty("DB_NAME", "");
         String dbPort = ApplicationContext.getProperty("DB_PORT", "");
-        String connectionLifetime = ApplicationContext.getProperty("DB_MAX_LIFETIME", "1800000");
+        long connectionLifetime =
+                parseLong(ApplicationContext.getProperty("DB_MAX_LIFETIME", "1800000"));
```

Comment:
Let's not static import a method from a class.

```suggestion
                Long.parseLong(ApplicationContext.getProperty("DB_MAX_LIFETIME", "1800000"));
```

Reply:
Fixed.